pages/vi/nextcloud-tor.md

```diff
@@ -12,7 +12,10 @@
 
 ## Prepare Treehouses image
 
-Download the latest treehouses image from http://download.treehouses.io, then use [balenaEtcher](https://etcher.io) to flash the image onto your SD card.  
+Download the latest treehouses image from http://download.treehouses.io, then use [balenaEtcher](https://etcher.io) to flash the image onto your SD card.
+
+| WARNING: Use a new SD Card for these steps; otherwise you will encounter errors.|
+| --- |
```

Comment:
Placement within a table removes the keyword rendering...
`Warning: ` by itself would render the following text into an orange block element, however, that visual rendering gets ignored when its within a table.

If you want it in a table, there some HTML inline styling which can do it. But it might be easier to remove the `|` table styling in order to have the "warning" style to take affect.